src/content/en/tools/puppeteer/debugging.md

```diff
@@ -0,0 +1,13 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Helpful tips for debugging Puppeteer
+
+{# wf_updated_on: 2018-01-10 #}
+{# wf_published_on: 2018-01-10 #}
+{# wf_blink_components: Internals>Headless #}
+
+{% include "web/tools/puppeteer/_shared/styles.html" %}
```

Comment:
By hiding the `<h1>`, the weight of the `<h2>` looks weird. Instead, can you hide the first h2, and leave the h1?

![screen shot 2018-01-12 at 11 01 42 am](https://user-images.githubusercontent.com/718038/34883589-02b86b16-f788-11e7-8136-6ea084845aa9.png)

Reply:
Yea this was a hack for now. Not all of the .md files even have a consistent `<h2>` so it's hard to generalize fixing these up. I'm ok with it being a little funky for now.